zuul.d/projects.yaml

```diff
@@ -49,6 +49,18 @@
       - publish-to-galaxy
       - publish-to-automation-hub
 
+- project:
+    name: github.com/ansible-collections/ansible.security
+    default-branch: main
+    merge-mode: squash-merge
+    templates:
+      - system-required
+      - integrated-queue
+      - ansible-collections-ansible-security
+      - ansible-python-jobs
+      - publish-to-galaxy
+      - publish-to-automation-hub
+
```

Comment:
This needs to be in a 2nd commit. Lets remove

Reply:
@pabelanger updated